macros/src/main/scala/MacroCompiler.scala

```diff
@@ -697,7 +702,8 @@ object MacroCompiler extends App {
     .map { case (_, cmd, description) => s"    $cmd: $description" }
   val usage: String = (Seq(
     "Options:",
-    "  -m, --macro-conf: The set of macros to compile in firrtl-generated conf format",
+    "  -m, --macro-conf: The set of macros to compile in firrtl-generated conf format (exclusive with -j)",
+    "  -j, --macro-mdf: The set of macros to compile in MDF JSON format (exclusive with -m)",
```

Comment:
Does `-c, --macro-conf` and `-m, --macro-mdf` work by any chance?

Reply:
`-c` is taken